### DISCUSSION TITLE
Docker Install : Error response from daemon: manifest unknown

### DISCUSSION THREAD

**Discussion by ai-dijon** [akeeba/panopticon#350] [Q&A (Community Support)] [answered]
Hello,
I try to install panopticon via Docker.
When i run "docker compose up -d" i get the error : "Error response from daemon: manifest unknown"
What am i missing ?

Thank you :)

**Comment by nikosdion** [comment, marked as answer]
Can you please try again?

It looks like the image push to the GitHub Container Registry was broken because of wrong token scopes. I fixed that and pushed the image for 1.0.4 manually.

**Comment by ai-dijon** [comment]
Fixed !

Thank you very much for your time and for the marvelous tools you give us

with love

**Reply by nikosdion** [reply]
The link is irrelevant. 

The actual problem is that even though the package was set up to inherit permissions from the repository, when you create a package on GitHub it makes in _internal_ by default. You have to go to the organisation's Packages page, click on the package, find its kinda hidden Settings link, and find the kinda hidden link to change its visibility. Groan…

I have now told GitHub that yes, indeed, I am definitely certain this is a public package. Can you pull it now?

**Reply by ai-dijon** [reply]
edited my comment sorry
